plasmapy/utils/formatting/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Tools for formatting strings, including for error messages."""
```

Comment:
I would argue that this sub-directory could/should be decomposed into a single `formatting.py` file, much like we did with `roman.py`.  I'd back off this if you see a significant expansion of the functionality contained within this sub-directory, but I do not see that at the moment.

I also don't think the name `formatting` quite represents what this functionality is doing.  Something like `repr` would better reflect what the functionality is doing.  Obviously, `repr` would be a poor name due to naming conflicts. ...`code_repr`, `obj2str`, `obj2repr`, `obj2code`, `obj2codestr` ...idk, none of these seem to be elegant or representative.

Reply:
Good idea...I'll move this into a single file.  I can't think of anything else what would go into `formatting` at the moment, though I expect some things here and there will come up.  

I'll think about the name...thus far some variation of `code_repr` seems best.